price.go

```diff
@@ -99,16 +101,14 @@ type PriceTierParams struct {
 	FlatAmountDecimal *float64 `form:"flat_amount_decimal,high_precision"`
 	UnitAmount        *int64   `form:"unit_amount"`
 	UnitAmountDecimal *float64 `form:"unit_amount_decimal,high_precision"`
-	UpTo              *int64   `form:"-"` // handled in custom AppendTo
-	UpToInf           *bool    `form:"-"` // handled in custom AppendTo
+	UpTo              *int64   `form:"up_to"`
+	UpToInf           *bool    `form:"-"` // See custom AppendTo
 }
 
-// AppendTo implements custom up_to serialisation logic for tiers configuration
+// AppendTo implements custom encoding logic for PriceTierParams.
 func (p *PriceTierParams) AppendTo(body *form.Values, keyParts []string) {
 	if BoolValue(p.UpToInf) {
 		body.Add(form.FormatKey(append(keyParts, "up_to")), "inf")
-	} else {
```

Comment:
This clause appears to be redundant with the default encoder for int64: https://github.com/stripe/stripe-go/blob/master/form/form.go#L319-L325

Reply:
i confirmed we have a test for it in https://github.com/stripe/stripe-go/blob/master/price_test.go#L52